app/models/setting/provisioning.rb

```diff
@@ -60,7 +60,7 @@ def self.default_settings
       self.set('dns_conflict_timeout', N_("Timeout for DNS conflict validation (in seconds)"), 3, N_('DNS conflict timeout')),
       self.set('clean_up_failed_deployment', N_("Foreman will delete virtual machine if provisioning script ends with non zero exit code"), true, N_('Clean up failed deployment')),
       self.set('name_generator_type', N_("Random gives unique names, MAC-based are longer but stable (and only works with bare-metal)"), 'Random-based', N_("Type of name generator"), nil, {:collection => Proc.new {NameGenerator::GENERATOR_TYPES} }),
-      self.set('default_pxe_item_global', N_("Default PXE (PXELinux, PXEGrub, PXEGrub2) menu item in global template - 'local', 'discovery' or custom"), 'local', N_("Default PXE global template entry")),
+      self.set('default_pxe_item_global', N_("Default PXE (PXELinux, PXEGrub, PXEGrub2, iPXE) menu item in global template - 'local', 'discovery' or custom"), 'local', N_("Default PXE global template entry")),
```

Comment:
This conflicts with other PR I filed this morning, I am getting rid of the list of PXE loaders in this description text anyway. Can you simply remove this hunk? Sorry about that.